src/main/java/com/hierynomus/smbj/share/DiskShare.java

```diff
@@ -260,6 +260,35 @@ public void mkdir(String path) throws SMBApiException {
         fileHandle.close();
     }
 
+    /**
+     * Create a set of nested sub-directories in the given path, for example, 2345 \ 3456 \ 4453 \ 123123.txt
+     */
+    public void mkdirs(String path) throws SMBApiException {
```

Comment:
Move this to the `SmbFiles` utility class. The `mkdirs` is not a part of the protocol directly, and now has too much path handling magic. Also do not commit commented out code.

Reply:
Mkdirs procedure has been moved to the SmbFiles utility class, comments are removed.